google-cloud-datastore/lib/google/cloud/datastore/service.rb

```diff
@@ -140,9 +142,25 @@ def generate_read_options consistency, transaction
             return Google::Cloud::Datastore::V1::ReadOptions.new(
               transaction: transaction
             )
+          elsif read_time
+            return Google::Cloud::Datastore::V1::ReadOptions.new(
+              read_time: read_time_to_timestamp(read_time)
+            )
           end
           nil
         end
+
+        def read_time_to_timestamp time
+          return nil if time.nil?
```

Comment:
Isn't the type of `read_time` , `Time` only?

Reply:
User input of `read_time` is `Time`. Data type of `read_time` in request is `Protobuf::Timestamp`.